Behavior/Onboarding/InterestingModules.swift

```diff
@@ -30,11 +30,11 @@ struct InterestingModules: View {
                 SequentialOnboardingView.Content(
                     title: "INTERESTING_MODULES_AREA3_TITLE",
                     description: "INTERESTING_MODULES_AREA3_DESCRIPTION"
-                ),
-                SequentialOnboardingView.Content(
-                    title: "INTERESTING_MODULES_AREA4_TITLE",
-                    description: "INTERESTING_MODULES_AREA4_DESCRIPTION"
                 )
+//                SequentialOnboardingView.Content(
```

Comment:
Please remove unnecessary code if not needed. Otherwise, feel free to add more content to the onboarding view!